### PR TITLE
Core: fixed TF linking issue, closes #199

### LINKED ISSUE #199: TF Linking broken
Scenario: Link two transfer functions. Create a new TF point in one of the TFs. Manipulate the same TF point in the other TF. This will not affect the second TF!

### PR DESCRIPTION
**Pull request by martinfalk**

### CODE REVIEW

**Review by r-englund — APPROVED** [review]

File: src/core/datastructures/tfprimitive.cpp [hunk 1]
```diff
@@ -37,6 +37,10 @@ TFPrimitive::TFPrimitive(double pos, const vec4& color) : data_({pos, color}) {}
 
 TFPrimitive::TFPrimitive(const TFPrimitiveData& data) : data_(data) {}
 
+TFPrimitive::TFPrimitive(const TFPrimitive& rhs) : data_(rhs.data_) {}
```
Comment (petersteneteg) [inline src/core/datastructures/tfprimitive.cpp:40]: Is this not equal to default? Maybe comment on why in that case?

**Review by petersteneteg — COMMENTED** [review]
?

File: src/core/datastructures/tfprimitive.cpp (hunk 1 above)
Comment (martinfalk) [reply src/core/datastructures/tfprimitive.cpp:40]: necessary because the constructor of Observable is not called!

Comment (martinfalk) [reply src/core/datastructures/tfprimitive.cpp:40]: see #199

Comment (petersteneteg) [reply src/core/datastructures/tfprimitive.cpp:40]: There should really be a comment about that...

Comment (martinfalk) [reply src/core/datastructures/tfprimitive.cpp:40]: added comment in 63f25e5

**Pull request merged**